src/_includes/page/sub-nav.html

```diff
@@ -12,9 +12,9 @@ <h6 class="bold">Customize</h6>
             <a href="http://pixijs.io/customize/" target="_blank"><span>Customize</span></a>
         </div>
         <div class="col">
-            <h6 class="bold">Game Devs Forum</h6>
-            <p>Game specific support for PixiJS courtesy of HTML5 Game Dev.</p>
-            <a href="http://www.html5gamedevs.com/forum/15-pixijs/" target="_blank"><span>Explore</span></a>
+            <h6 class="bold">Discord</h6>
+            <p>Join our PixiJS Discord community! Chat to the team, get support, make friends!</p>
+            <a href="https://discord.gg/AZhDQeS5BF" target="_blank"><span>Explore</span></a>
```

Comment:
Where'd you get that link? I grabbed the one from the README which I think @miltoncandelero said doesn't expire.
```suggestion
            <a href="https://discord.gg/QrnxmQUPGV" target="_blank"><span>Explore</span></a>
```

Reply:
oh i created a new one! this way we can track (via discord server insights) who comes from the site vs who comes from the readme!